packages/rules-typescript/src/tests/src-detector.test.ts

```diff
@@ -23,3 +24,17 @@ test("Should fail if `src` detected in imports", async () => {
   const result = sut.checkImportPaths(asBollFile("a"), importPaths);
   assert.strictEqual(ResultStatus.failure, result[0].status);
 });
+
+test("Should fail if references to node_modules exist in source code", async () => {
```

Comment:
Should this say: "Should fail if references to **src** exist in source code"?

Reply:
Updated